Codeinwp/neve
# enh: improvements to the Neve Options Dashboard Page

### Summary
Enhancements to the dashboard options page 


### Will affect visual aspect of the product

NO

### Test instructions


- Dashboard should work according to specified enhancements in the issue. 
- Some things have been adapted (the facebook button is not a card but a button in another card)
- All links should be functional and use proper UTM parameters
- Dashboard should work as before
- WhiteLabel module from pro should remove the same things as before.


Closes https://github.com/Codeinwp/neve-pro-addon/issues/929
